src/nginx/t/transcoding_ignore_unknown_fields.t

```diff
@@ -103,6 +103,18 @@ EOF
 ok(ApiManager::verify_http_json_response($response,
     {'id'=>'100', 'theme' => 'Classics'}), "The shelf was created successfully.");
 
+# Test unknown field in response. ResponseMessage UnknownBook has a field "unknown"
+# which is known to gRPC server, but not to ESP.
+# Expected result is: only that field is skipped, other fields should be fine.
+my $response1 = ApiManager::http_get($NginxPort,'/unknownbook?key=this-is-an-api-key');
+
+# With bug in https://github.com/cloudendpoints/esp/issues/795, the first field
+# after the uknown is discarded. Once the bug is fixed, use follow correct result.
```

Comment:
Without the fix,  the first field "known_int" is discarded.  With the fix,  all fields are there